### DISCUSSION TITLE
Installation Guide Phalcon v4.2 + PHP 8.0 ?

### DISCUSSION THREAD

**Discussion by coffeeyoghurt** [phalcon/cphalcon#16087] [Q&A]
Is there an installation guide available for Phalcon v4.2 + PHP 8.0? Or if not; what is necessary to get this running on a new server? 

Thanks,

**Comment by temuri416** [comment]
This worked for me once:

```cd /root/install && rm -rf cphalcon && git clone -b 4.2.x https://github.com/phalcon/cphalcon.git && cd cphalcon && wget http://github.com/phalcon/zephir/releases/download/0.15.2/zephir.phar && chmod +x zephir.phar && mv zephir.phar /usr/sbin && zephir.phar fullclean && zephir.phar build && cd $(php-config --extension-dir) && mv phalcon.so phalcon-4.2.x.so```
